docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```
@@ -204,9 +205,19 @@ subquestion: |
   **Pause here.** Ask your admin whether your organization or repository already has the {right GitHub secrets}.
   % endif
 fields:
-  # Always
+  # If is org admin
   - Has someone already set up the repository or organization secrets?: already_has_secrets
+    show if:
+      code: |
+        is_org_admin
     datatype: yesnoradio
+  # If NOT org admin
+  - Has someone already set up the repository's secrets?: already_has_secrets
+    show if:
+      code: |
+        is_org_admin is False
+    datatype: yesnoradio
```

If all we're doing is changing the label based on `is_org_admin`, we can split it into `label` and `field` attributes and use mako for the label:

```suggestion
  # If is org admin
  - label: |
      % if is_org_admin:
      Has someone already set up the repository or organization secrets?
      % else:
      Has someone already set up the repository's secrets?
      % endif
    field: already_has_secrets
    datatype: yesnoradio
```

---

It occurs to me now that if someone is not an org admin, someone else could have already set up their organization's secrets. So maybe we just don't need the `if` statement? Or have I lost the context somehow?

This will be a great change if we do need the `if` statement, though. I didn't remember that you could split up the label and the field name that way, thanks!

---

> It occurs to me now that if someone is not an org admin, someone else could have already set up their organization's secrets. So maybe we just don't need the if statement? Or have I lost the context somehow?

From [the relevant code block](https://github.com/plocket/docassemble-ALAutomatedTestingTests/blob/f2aa2ab12af5583e0b63b3cc87122452a50638ed/docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml#L65-L76), it looks like we only make or override org secrets if they are an admin. So it doesn't make sense to ask specifically about org secrets if they aren't an admin, because the next question is asking if they want to override those secrets. The implicit question is "has someone already setup org secrets? 
-> if they haven't, go ask them to, if they have, you can't override them, so just keep going", but that adds a descent bit of complexity.

---

Will have to make this into an issue.